manager/src/test/java/com/redhat/service/bridge/manager/ProcessorServiceTest.java

```diff
@@ -365,20 +365,23 @@ void createConnector() {
         BaseAction slackAction = createSlackAction();
         ProcessorRequest processorRequest = new ProcessorRequest("ManagedConnectorProcessor", slackAction);
 
-        when(connectorsApiClient.createConnector(any())).thenReturn(new Connector());
+        Connector connector = new Connector();
+
+        when(connectorsApiClient.getConnector(any())).thenReturn(connector);
+        when(connectorsApiClient.createConnector(any())).thenReturn(connector);
         when(rhoasService.createTopicAndGrantAccessFor(anyString(), any())).thenReturn(new Topic());
 
         Processor processor = processorService.createProcessor(b.getId(), b.getCustomerId(), processorRequest);
 
         await().atMost(5, SECONDS).untilAsserted(() -> {
-            ConnectorEntity connector = connectorsDAO.findByProcessorIdAndName(processor.getId(),
+            ConnectorEntity connectorEntity = connectorsDAO.findByProcessorIdAndName(processor.getId(),
```

Comment:
`ConnectorEntity` is just to differentiate the type from the MC API's `Connector` so I think it's safe to use it as a variable name - anyway both are perfectly fine